vnc_auto.html

```diff
@@ -80,7 +80,21 @@
                            "jsunzip.js", "rfb.js", "keysym.js"]);
 
         var rfb;
+        var resizeTimeout;
 
+
+        function UIresize() {
+            var innerWidth = window.innerWidth;
+            var innerHeight = window.innerHeight;
+            var controlBarHeight = $D('noVNC_status_bar').scrollHeight;
```

Comment:
Should this really be scrollHeight (which would count any non-visible content)?  Perhaps `offsetHeight` instead (rendered height including padding and borders.